Sources/Types.swift

```diff
@@ -194,6 +194,13 @@ public enum FitType {
     case vertically
 }
 
+@objc public enum ViewFilter: Int {
+    /// No filter, use all views
+    case none
+    /// Consider only views where `hidden` is false
```

Comment:
```suggestion
    /// Consider only visible views (isHidden is false or alpha is > 0)
```

Reply:
I believe 

>isHidden is false or alpha is > 0

should be:

isHidden is false `and` alpha is > 0